src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -1429,22 +1409,7 @@ public PoliticalActionAttachment getPoliticalActionChoice(final PlayerID player,
     }
     m_messageAndDialogThreadPool.waitForAll();
     m_actionButtons.changeToPolitics(player);
-    if (!SwingUtilities.isEventDispatchThread()) {
-      try {
-        SwingUtilities.invokeAndWait(new Runnable() {
-          @Override
-          public void run() {
-            requestFocusInWindow();
-            transferFocus();
-          }
-        });
-      } catch (final Exception e) {
-        e.printStackTrace();
-      }
-    } else {
-      requestFocusInWindow();
-      transferFocus();
-    }
+    requestWindowFocus();
```

Comment:
Do we not need the invokeAndWait here? Also looks like transferFocus() was removed?